portal-impl/src/com/liferay/portlet/documentlibrary/service.xml

```diff
@@ -201,12 +201,14 @@
 		<finder name="GroupId" return-type="Collection">
 			<finder-column arrayable-operator="OR" name="groupId" />
 		</finder>
-		<finder name="G_DDI" return-type="DLFileEntryType" unique="true">
+		<finder name="G_C_DDI" return-type="DLFileEntryType" unique="true">
 			<finder-column name="groupId" />
+			<finder-column name="companyId" />
 			<finder-column name="dataDefinitionId" />
 		</finder>
-		<finder name="G_F" return-type="DLFileEntryType" unique="true">
+		<finder name="G_C_F" return-type="DLFileEntryType" unique="true">
 			<finder-column name="groupId" />
+			<finder-column name="companyId" />
 			<finder-column name="fileEntryTypeKey" />
```

Comment:
Shouldn't the `companyId` precede the `groupId`?

Reply:
I thought the same, but apparently the SF rules have changed (this new order is enforced automatically).

Reply:
AFAIK is because group is before company on the declaration of the entity (line 176)